src/collections/Hero.tsx

```diff
@@ -92,8 +97,15 @@ export const Hero = ({
   const theme = !bgColor || bgColor === 'transparent' ? 'light' : lightOrDark(bgColor)
   const verPos = getVerPos(verPosVal)
   const horPos = getHorPos(horPosVal)
-  const bgImageOutput = bgType === 'image' && bgImage ? bgImage.url : bgType === 'video' && bgVideoFallback ? bgVideoFallback.url : ''
-  const parsedInfo = nonThrowingJsonParse(bgVideo?.info)
+
+  const parsedVideoInfos = nonThrowingJsonParse(bgVideo?.info)
+  const parsedImageInfos =
+    bgType === 'image' && bgImage
+      ? nonThrowingJsonParse(bgImage?.info)
+      : bgType === 'video' && bgVideoFallback && nonThrowingJsonParse(bgVideoFallback?.info)
+  const bgImageOutput = bgType === 'image' && bgImage ? bgImage.url : bgType === 'video' && bgVideoFallback ? bgVideoFallback.url : null
```

Comment:
Inverting the order of lines of code makes it look like a code change. When possible, keep the existing order to make it easier to spot the actual changes